src/operation/emulation_model.jl

```diff
@@ -463,173 +453,109 @@ Default solve method for an Emulation model used inside of a Simulation. Solves
-    container::OptimizationContainer,
-    execution_count,
-    exports,
-)
-    # This needs a new implementation that might be similar to DecisionModel
-    #     problem_name = get_name(model)
-    #     if exports !== nothing
-    #         exports_path = joinpath(exports[:exports_path], "variables")
-    #         mkpath(exports_path)
-    #     end
-    #
-    #     for (key, variable) in get_variables(container)
-    #         write_result!(
-    #             store,
-    #             problem_name,
-    #             STORE_CONTAINER_VARIABLES,
-    #             key,
-    #             timestamp,
-    #             variable,
-    #         )
-    #
-    #         if exports !== nothing &&
-    #            should_export_variable(exports[:exports], timestamp, problem_name, key)
-    #             horizon = exports[:horizon]
-    #             resolution = exports[:resolution]
-    #             file_type = exports[:file_type]
-    #             df = axis_array_to_dataframe(variable)
-    #             time_col = range(timestamp, length = horizon, step = resolution)
-    #             DataFrames.insertcols!(df, 1, :DateTime => time_col)
-    #             export_result(file_type, exports_path, key, timestamp, df)
-    #         end
-    #     end
+function _write_model_variable_results!(store, container, execution)
+    for (key, variable) in get_variables(container)
+        write_result!(store, STORE_CONTAINER_VARIABLES, key, execution, variable)
+    end
 end
 
-function _write_model_aux_variable_results!(store, container, execution, exports)
-    # This needs a new implementation that might be similar to DecisionModel
-    #     problem_name = get_name(model)
-    #     if exports !== nothing
-    #         exports_path = joinpath(exports[:exports_path], "variables")
-    #         mkpath(exports_path)
-    #     end
-    #
-    #     for (key, variable) in get_aux_variables(container)
-    #         write_result!(
-    #             store,
-    #             problem_name,
-    #             STORE_CONTAINER_VARIABLES,
-    #             key,
-    #             timestamp,
-    #             variable,
-    #         )
-    #
-    #         if exports !== nothing &&
-    #            should_export_variable(exports[:exports], timestamp, problem_name, key)
-    #             horizon = exports[:horizon]
-    #             resolution = exports[:resolution]
-    #             file_type = exports[:file_type]
-    #             df = axis_array_to_dataframe(variable)
-    #             time_col = range(timestamp, length = horizon, step = resolution)
-    #             DataFrames.insertcols!(df, 1, :DateTime => time_col)
-    #             export_result(file_type, exports_path, key, timestamp, df)
-    #         end
-    #     end
+function _write_model_aux_variable_results!(store, container, execution)
+    for (key, variable) in get_aux_variables(container)
+        write_result!(store, STORE_CONTAINER_AUX_VARIABLES, key, execution, variable)
```

Comment:
Can you confirm that aux_variables will always be stored independently of variables? We have some inconsistency.

Reply:
This is something we need to decide on. I prefer them to be stored separately for clarity